CMSgov/design-system
# [WNMGDS-1901] Remove util class for Drawer close button, update snapshots

## Summary

[JIRA](https://jira.cms.gov/browse/WNMGDS-1901)

Removed utility class from Drawer close button.

Prior to the Button design update, this close button had a white background on the core theme. When the Buttons were updated, the default background on core buttons was transparent. This meant the Drawer close button now appeared gray (the color of the Drawer header background). So to maintain that appearance, a white background util class was added - but didn't account for the behavior of this button on other themes.

Removing this util fixes the reported hover bug for child themes, but will result in the core close button being gray, which I guess is cool? 

## How to test

1. Pull this branch
2. Build Storybook
3. View Drawer story in all three themes
4. Notice the hover state works now

### If this is a change to code:


### If this is a change to documentation:
